src/Containers/BikeMap/BikeMap.js

```diff
@@ -52,6 +54,15 @@ export class BikeMap extends Component {
             center={[30, 0]}
             zoom='3'>
             {this.markersToShow()}
+            {/* MapSearch is my withLeaflet wrapper on ReactLeafletZoomIndicator - I also tried wrapping it directly in this file */}
+            <MapSearch />
```

Comment:
MapSearch is my withLeaflet wrapper on ReactLeafletZoomIndicator - I also tried wrapping it directly in this file

Reply:
ReactLeafletZoomIndicator is undefined, since it's not part of the leaflet search react library